tests/unit/test_agent_skill.py

```diff
@@ -22,12 +23,21 @@
 )
 
 
+# current file needs to be reset after each test or some
+# tests will fail because they are dependent on the current
+# file being set correctly
+@pytest.fixture(autouse=True)
+def reset_current_file():
+    init_file()
```

Comment:
```suggestion
    from opendevin.runtime.plugins.agent_skills import agentskills
    agentskills.CURRENT_FILE = None
    yield
    agentskills.CURRENT_FILE = None
```
What do you think about something like this? It seems to work on my machine

Reply:
Ohhh... I think I forgot to prefix it with "agentskills.", doh! 🫨 
I'll give it a go an amend this PR!
Thanks!!